deephaven_ipywidgets/_version.py

```diff
@@ -4,5 +4,5 @@
 # Copyright (c) Deephaven Data Labs LLC.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 1, 0)
```

Comment:
Python dev versions should be numbered: `0.2.0.dev0`, etc.

Reply:
https://peps.python.org/pep-0440/#public-version-identifiers

Reply:
Yea I thought so too... I was basing this off the template which just had `'dev'` previously: https://github.com/deephaven/deephaven-ipywidgets/blob/5622157a606ca332b3514833e0ae7563d5d44251/deephaven_ipywidgets/_version.py#L7

There's been a couple things wrong with this template. I'll change it to dev0.